tests/loadtests/vkloadtests/VulkanLoadTestSample.cpp

```diff
@@ -65,11 +65,11 @@ VulkanLoadTestSample::loadMesh(std::string filename,
 vk::PipelineShaderStageCreateInfo
 VulkanLoadTestSample::loadShader(std::string filename,
                                  vk::ShaderStageFlagBits stage,
-                                 std::string modname)
+                                 const char* modname)
 {
     vk::PipelineShaderStageCreateInfo shaderStage({}, stage);
     shaderStage.module = vkctx.loadShader(filename);
-    shaderStage.pName = modname.c_str();
+    shaderStage.pName = modname;
```

Comment:
And this.

Reply:
This pointer can't be returned because it's owned by an object on the stack.